backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/helper/ImplicitSpecializationSwitch.java

```diff
@@ -239,6 +245,26 @@ private Feature computeSourceFeature(SuccessionAsUsage successionAsUsage) {
         return null;
     }
 
+    private Feature computeTargetFeature(SuccessionAsUsage successionAsUsage) {
```

Comment:
Please put private methods at the end. This is valid for this PR and every other PR in the future.

Reply:
Done.